src/components/Text/Text.tsx

```diff
@@ -22,7 +23,8 @@ export type Props = {
   /**
    * Controls whether to render text inline (defaults to "p");
    */
-  as?: 'p' | 'span';
+  as?: 'p' | 'span' | 'div';
+  capLinelength?: boolean;
   children: React.ReactNode;
   className?: string;
   variant?: Variant;
```

Comment:
I can't leave a comment on the `spacing` prop line, but I wonder if we should consider renaming that to `spacingBottom` or something since `spacing` could now be misinterpreted to mean the spacing between sections inside the text passage.

Reply:
That does make sense... will rename to `spacingBottom`

Reply:
`spacingBottom` also makes more sense in the text-passage use case

Reply:
Is there a different prop that controls spacing between sections? If not, as long as we document clearly I don't think people will be too confused. If we'd prefer more clarity (and don't mind the codemod -- looks like there are only ~49 usages) maybe we fully rename it to `marginBottom` for max clarity?

Reply:
There is no prop that controls spacing between sections, spacing between sections is specific to the child element. Personally I would like to keep it as `spacing` for prop consistency but wouldn't mind `spacingBottom` or `marginBottom`

Reply:
haha ya sorry sneaky edit, I decided that i'm indifferent between `spacingBottom` and `marginBottom` if we rename

Reply:
Do you mind if I revert it back to `spacing` and also add a comment such as `Adds margin bottom spacing to the <Text> component`

Reply:
consistency is certainly valuable 🙂 I may err on the side of not introducing a breaking change unless we have proof people don't like the old way. hopefully with CSS untangling we can remove this prop altogether 🤞 but ultimately up to you!

Reply:
Made back to `spacing` for consistency and not introducing breaking change, thanks all